app-emacs/hydra/hydra-0.15.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+NEED_EMACS="24.3"
+
+inherit elisp
+
+DESCRIPTION="A package for GNU Emacs that can be used to tie related commands \
```

Comment:
I'm surprised pkgcheck doesn't warn about this -- it's too [long](https://devmanual.gentoo.org/ebuild-writing/variables/index.html#ebuild-defined-variables)! If pkgcheck isn't warning about it, I think we should file a bug.

We can use <longdescription> in metadata.xml for longer stuff like this. I think it's discouraged / forbidden(?) to use line continuations in DESCRIPTION.